.zenodo.json

```
@@ -6,7 +6,166 @@
     "brain imaging data structure",
     "neuroscience",
     "neuroimaging",
-    "neuroinformatics",
+    "neuroinformatics"
+  ],
+  "creators": [
+    {
+      "affiliation": "Stanford University",
+      "name": "Blair, Ross",
+      "orcid": "0000-0003-3007-1056"
+    },
+    {
+      "name": "Michael, Zack"
+    },
+    {
+      "affiliation": "Google",
+      "name": "Gorgolewski, Krzysztof J.",
+      "orcid": "0000-0003-3321-7583"
+    },
+    {
+      "affiliation": "Squishymedia",
+      "name": "Hardcastle, Nell",
+      "orcid": "0000-0002-3837-0707"
+    },
+    {
+      "name": "Hobson-Lowther, Teal"
+    },
+    {
+      "name": "Nishikawa, David"
+    },
+    {
+      "name": "Bhogawar, Suyash",
+      "affiliation": "Northern California Institute for Research and Education, San Francisco CA"
+    },
+    {
+      "affiliation": "Center for Adaptive Rationality, Max Planck Institute for Human Development, Berlin, Germany",
+      "name": "Appelhoff, Stefan",
+      "orcid": "0000-0001-8002-0877"
+    },
+    {
+      "affiliation": "AA Martinos Center for Biomedical Imaging, USA",
+      "name": "Jas, Mainak",
+      "orcid": "0000-0002-3199-9027"
+    },
+    {
+      "name": "Grass, Brian"
+    },
+    {
+      "affiliation": "Stanford University",
+      "name": "Markiewicz, Christopher J.",
+      "orcid": "0000-0002-6533-164X"
+    },
+    {
+      "affiliation": "Berkeley Institute for Data Science; University of California at Berkeley",
+      "name": "Holdgraf, Chris",
+      "orcid": "0000-0002-2391-0678"
+    },
+    {
+      "name": "Jones, Alexander",
+      "affiliation": "University of Texas at San Antonio",
+      "orcid": "0000-0003-2067-3862"
+    },
+    {
+      "name": "Goyal, Rohan"
+    },
+    {
+      "name": "Oostenveld, Robert",
+      "affiliation": "Radboud University",
+      "orcid": "0000-0002-1974-1293"
+    },
+    {
+      "name": "Noack, Gregory"
+    },
+    {
+      "name": "Triplett, William"
+    },
+    {
+      "name": "Naveau, Mikaël",
+      "affiliation": "CYCERON imaging plateform, Caen, France",
+      "orcid": "https://orcid.org/0000-0002-4685-0057"
+    },
+    {
+      "name": "Zito, Matthew",
+      "orcid": "0000-0003-2704-0742"
+    },
+    {
+      "name": "Durnez, Joke"
+    },
+    {
+      "name": "Traut, Nicolas"
+    },
+    {
+      "name": "Sethi, Parul"
+    },
+    {
+      "affiliation": "Dartmouth College",
+      "name": "Halchenko, Yaroslav O.",
+      "orcid": "0000-0003-3456-2493"
+    },
+    {
+      "name": "Papadopoulos, Dimitri"
```

@DimitriPapadopoulos 

```suggestion
      "affiliation": "CEA",
      "name": "Papadopoulos Orfanos, Dimitri",
      "orcid": "0000-0002-1242-8990"
```